Basic-Car-Maintenance.xcodeproj/project.pbxproj

```diff
@@ -72,7 +73,7 @@
 /* Begin PBXCopyFilesBuildPhase section */
 		FFDADF8F2ACD35A200DDEF79 /* Embed Foundation Extensions */ = {
 			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
+			buildActionMask = 12;
```

Comment:
Do you know what this is changing? If not, I'll just change it back

Reply:
I have no idea. I'm not sure how it even was changed.